### PR TITLE
Script to download image references from CircleCI machine

### LINKED ISSUE #63: Setup a script to generate and download image references from CircleCI
The visual regression references should be generated in the CI environment.

Currently, the way to go is to upload the code, run the CI with ssh and manually download the references using [scp](https://en.wikipedia.org/wiki/Secure_copy).

We need to improve this process. Some suggestions:

- Create a script that can be run on the CI which generates the references and creates a new PR/commit with them.
- Create a script to easy scp the references from the CI and manually add the new references to the repo

### PR DESCRIPTION
**Pull request by jesusbotella**
This PR includes the update references script. It updates the local image reference for styles testing.

### Process
- Retrieve all the image paths inside `airship/packages/styles/src/` folder located in the remote machine.
- Download them to  `.tmp_reference` folder in local machine via SCP.
- Replace the previous style tests reference with the new one.
- Remove `.tmp_reference` folder

### Usage
`npm run update-styles-reference -- {ip} {port}`

### Script Output
screen shot 2018-07-19 at 18 17 56

Closes #63.

### CODE REVIEW

File: DEVELOPERS.md [hunk 1]
```diff
@@ -5,16 +5,15 @@ We're using CircleCI as our Continuous Integration server. It currently runs sev
...
 Follow these manual steps to fix the problem, as snapshots generated at local dev and remote CircleCI machine may differ, and we want CircleCI ones to be the authoritative source:
 1. comment img deletion from the test code (`/airship/packages/styles/src/test.js` >> LIN 29: `//fs.unlinkSync(output);`)
 2. commit & push, to force new CircleCI run
 3. ssh connection to new docker machine generated for the job. See [ssh reference](https://circleci.com/docs/2.0/ssh-access-jobs/) for troubleshooting.
-4. download new image reference with **scp**:
-    - You must connect with your *github credentials* to the temporal machine for the job, using the IP AND port displayed by CircleCI Web console (let's assume ip is **52.90.13.2** and port is **64546**). To check the public key being used: `ssh -v -p 64546 52.90.13.2`
-    - And then copy with something like `scp -P 64546 52.90.13.2:airship/packages/styles/src/button/img/button.png ./reference.png`
-5. commit the new image
+4. download new image reference with our **npm script**:
+    - execute `npm run update-styles-reference -- {ip} {port}` with your SSH credentials in CircleCI
```
Comment (VictorVelarde) [inline DEVELOPERS.md]: ✨

File: package.json [hunk 2]
```diff
@@ -7,11 +7,12 @@
   "author": "Carto Dev Team",
   "license": "BSD-3-Clause",
   "devDependencies": {
-    "exquisite-sst": "^1.4.0",
+    "chalk": "^2.4.1",
```
Comment (IagoLast) [inline package.json]: we are using chalk and colors.js, we should chose one to reduce install time :)

Comment (jesusbotella) [reply package.json]: Oops! I didn't realize, let me change it!

**Comment by IagoLast** [comment]
We introduced in #63 an easier way to generate the screenshots

**Pull request closed without merging**